### DISCUSSION TITLE
Supressing 'done'

### DISCUSSION THREAD

**Discussion by rofo69** [BigBobbas/ESP32-S3-Box3-Custom-ESPHome#56] [General]
Firstly, I wanna say this firmware is miles beyond what I had hoped for, and finally gives me the ability to send back custom phrases to the speaker on the device. I have a super simple test that listens for a phrase, then returns a phrase. But the device then always finished off with the word 'done'. Can this be disabled in any way ? Naturally I don't want to mute the speaker, as I wouldn't then hear my custom phrase.

**Comment by BigBobbas** [comment]
HI ,
thank you for your support with the project.
I believe that this is something that is generated on the HA side of things. You may be able to add a custom response either by an automation response or by adding a responses.yaml - there may be some information to help [here](<https://www.home-assistant.io/voice_control/custom_sentences/>)
If not if you can post a little more detail with regards to the way that your test is implemented.

Best Wishes
Rob

**Reply by rofo69** [reply]
Seems the answer was actually trivial, and in the link you supplied.
Rather than responding with a Piper TTS action, you simply use an 'other action' and select 'set conversation response'.
